lib/Service/RecipeService.php

```diff
@@ -1130,61 +1098,25 @@ public function parseRecipeFile($file) {
 	 * @return File
 	 */
 	public function getRecipeImageFileByFolderId($id, $size = 'thumb') {
```

Comment:
```suggestion
	public function getRecipeImageFileByFolderId($id, $size = 'thumb'): File
	{
```

Reply:
I did not touch the Recipe Service class. But yes, that should be safe do to so far.